src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -82,4 +82,16 @@ public void verifyNoDuplicatePrefixesFor(Prefix... prefixes) throws ParseExcepti
     public int numberOfUniquePrefixes() {
         return this.argMultimap.size() - 1;
     }
+
+    /**
+     * Checks if the {@code ArgumentMultimap} contains a valid entry for the specified prefix.
+     *
+     * @param prefix the {@code Prefix} we want to check for in the argument map.
+     * @return {@code True} if the prefix is present with exactly one non-empty argument;
+     *         {@code False} otherwise.
+     */
+    public boolean containsValidSinglePrefix(Prefix prefix) {
```

Comment:
Good implementation of method, but I think name of method may be misleading. Maybe something like isValidSinglePrefixArgument because the parameter is prefix